### PR TITLE
Fix saving viewflow tasks with formsets

### PR DESCRIPTION
**Pull request by Ashlett**
Fixes https://sentry.io/thermondo/thermondo-backend/issues/410339251/

#### How to test

Go to the backend and assign yourself a task with formset (e.g. Technical check / Create article list). Try to save it. The above error should not occur.

### CODE REVIEW

File: viewflow_extensions/views.py [hunk 1]
```diff
@@ -61,3 +65,8 @@ def get_success_url(self):
         if self._save:
             return self.request.get_full_path()
         return super().get_success_url()
+
+
+def _make_fields_not_required(form):
```
Comment (codingjoe) [inline viewflow_extensions/views.py]: No private functions ;) Make it public write a docstring, but it in the proper package ;)

Comment (codingjoe) [reply viewflow_extensions/views.py]: and write a unit test :P

Comment (Ashlett) [reply viewflow_extensions/views.py]: what do you mean "but it in the proper package"?

Comment (codingjoe) [reply viewflow_extensions/views.py]: sorry I meant module. I would recommend `forms` or `utils`.

**Pull request merged**